react-app-drawer-component/src/AppDrawer.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+
+function AppDrawer() {
+  const [open, setOpen] = useState(false);
+
+  function handleClick() {
+    setOpen((prev) => !prev);
+  }
+
+  const hiddenBurger = open ? '' : 'hidden';
+  const navModal = open ? 'nav-modal' : '';
+
+  function hiddenSwap() {
```

Comment:
You could calculate this as a variable instead of needing a function. (Variables are easier to read and work with, so they're usually the best choice.) Something like:
`const hiddenSwap = hiddenBurger ? '' : 'hidden';`